equinix/resource_fabric_service_profile.go

```diff
@@ -863,11 +863,11 @@ func resourceServiceProfilesSearchRequest(ctx context.Context, d *schema.Resourc
 	return setFabricServiceProfilesListMap(d, serviceProfiles)
 }
 
-func customFieldFabricSpToTerra(customFieldl []v4.CustomField) []map[string]interface{} {
+func customFieldFabricSpToTerra(customFieldl []v4.CustomField) []interface{} {
 	if customFieldl == nil {
 		return nil
 	}
-	mappedCustomFieldl := make([]map[string]interface{}, len(customFieldl))
+	mappedCustomFieldl := make([]interface{}, len(customFieldl))
```

Comment:
Use `s` as the suffix instead of `s` to indicate multiple items. Lower cognitive overhead when reading the names.